rssfeed/models.py

```diff
@@ -17,37 +18,27 @@ def __unicode__(self):
         return self.title or self.xml_url
 
     def save(self, *args, **kwargs):
-        """Poll new Feed"""
+        """Poll new Feed """
         try:
             Feed.objects.get(xml_url=self.xml_url)
             super(Feed, self).save(*args, **kwargs)
         except Feed.DoesNotExist:
             super(Feed, self).save(*args, **kwargs)
             from rssfeed.utils import poll_feed
```

Comment:
You can't poll the feed in the save method. It's a blocking call. Fire off a celery task to update the object asynchronously.